source/js/components/carousel/carousel.js

```diff
@@ -1,10 +1,11 @@
-import Swiper, {
+import Swiper from "swiper";
+import {
   A11y,
   Autoplay,
   Pagination,
   Navigation,
   Keyboard,
-} from "swiper";
+} from "swiper/modules";
```

Comment:
This is due to [the changes](https://swiperjs.com/changelog#features-52) introduced in v7

<img width="682" alt="image" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2896608/cc3720ba-7eb5-432b-baed-08dcbbf5bb95">